web_external/layout/HeaderView.js

```diff
@@ -15,6 +15,7 @@ isic.views.LayoutHeaderView = isic.View.extend({
         this.render();
 
         girder.events.on('g:login', this.render, this);
+        girder.events.on('g:login-changed', this.render, this);
```

Comment:
FYI, you can bind to multiple events in the same call just by delimiting the event names spaces. Also, we should use `listenTo` in this sort of context, to make it safer when this view is destroyed (though the teardown method should handle it).

However, both things are already fixed in the 2.x branch, so it's not a big deal here.